interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -811,6 +811,17 @@ public AccessToken refreshAccessToken() throws IOException {
         authScope.contains(response.getOauthScope()));
   }
 
+  /** Sends an rpc to an unimplemented method on the server. **/
+  public void unimplementedMethod() {
+    UnimplementedServiceGrpc.UnimplementedServiceBlockingStub stub =
+        UnimplementedServiceGrpc.newBlockingStub(channel);
+    try {
+      stub.unimplementedCall(Empty.getDefaultInstance());
+    } catch (Throwable t) {
+      assertEquals(Status.UNIMPLEMENTED, Status.fromThrowable(t));
```

Comment:
Per the [spec](https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md#unimplemented_method), we should check the code and message.
```
assertEquals(Status.UNIMPLEMENTED.getCode(), Status.fromThrowable(t).getCode());
assertNull(Status.fromThrowable(t).getDescription());
```
I think the second check will fail with the current implementation, so we should either:
1. update the server side code to not append the description, or
2. remove this overstrict limit from the Spec and don't do such check.

@ejona86, what do you think?